libs/sdk-ui/src/base/localization/TranslationsCustomizationProvider/utils.ts

```diff
@@ -85,13 +85,19 @@ export const pickCorrectWording = (
     pickCorrectMetricWording(pickCorrectInsightWording(translations, settings), settings);
```

Comment:
Please don't forget to update the copyright accordingly.

Reply:
fixed